app/helpers/lookup_keys_helper.rb

```diff
@@ -79,12 +79,12 @@ def can_edit_params?
   def lookup_key_with_diagnostic(obj, lookup_key, lookup_value)
     value, matcher = value_matcher(obj, lookup_key)
     inherited_value = lookup_key.value_before_type_cast(value)
-    overridden  = inherited_value.present? || lookup_value.value.present?
```

Comment:
This returned `true` if one of them was `true` and `false` if one was set to `false` and the other was `nil`.